TanukiKit/Repositories.swift

```diff
@@ -5,60 +5,459 @@ import RequestKit
+    /**
+     Fetches the Projects which the authenticated user can see.
+     - parameter page: Current page for project pagination. `1` by default.
+     - parameter perPage: Number of projects per page. `100` by default.
+     - parameter archived: Limit by archived status. Default is `""`, set to `true` to enable.
+     - parameter visibility: Limit by visibility `public`, `internal` or `private`. Default is `""`
+     - parameter orderBy: Return projects ordered by `id`, `name`, `path`, `created_at`, `updated_at`, or `last_activity_at` fields. Default is `created_at`.
+     - parameter sort: Return projects sorted in asc or desc order. Default is `desc`.
+     - parameter search: Return list of authorized projects matching the search criteria. Default is `""`
+     - parameter simple: Return only the ID, URL, name, and path of each project. Default is `""`, set to `true` to enable.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+
+    // TODO: visibleProjects convertion.
+    public func visibleRepos(_ session: RequestKitURLSession = URLSession.shared, page: String = "1", perPage: String = "20", archived: String = "", visibility: String = "", orderBy: String = "created_at", sort: String = "desc", search: String = "", simple: String = "", completion: @escaping (_ response: Response<[Repository]>) -> Void) -> URLSessionDataTaskProtocol? {
+        let router = ProjectRouter.readVisibleProjects(configuration: configuration, page: page, perPage: perPage, archived: archived, visibility: visibility, orderBy: orderBy, sort: sort, search: search, simple: simple)
+        return router.loadJSON(session, expectedResultType: [[String: AnyObject]].self) { json, error in
+            if let error = error {
+                completion(Response.failure(error))
+            }
+
+            if let json = json {
+                let projects = json.map { Repository($0) }
+                completion(Response.success(projects))
+            }
+        }
+    }
+
+    /**
+     Fetches the Projects which are owned by the authenticated user.
+     - parameter page: Current page for project pagination. `1` by default.
+     - parameter perPage: Number of projects per page. `100` by default.
+     - parameter archived: Limit by archived status. Default is `""`, set to `true` to enable.
+     - parameter visibility: Limit by visibility `public`, `internal` or `private`. Default is `""`
+     - parameter orderBy: Return projects ordered by `id`, `name`, `path`, `created_at`, `updated_at`, or `last_activity_at` fields. Default is `created_at`.
+     - parameter sort: Return projects sorted in asc or desc order. Default is `desc`.
+     - parameter search: Return list of authorized projects matching the search criteria. Default is `""`
+     - parameter simple: Return only the ID, URL, name, and path of each project. Default is `""`, set to `true` to enable.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+
```

Comment:
Remove empty lines between documentation and method as Xcode does not pick it up otherwise.